russian/sleep/brushing/choice/blue-toothbrush.md

```diff
@@ -0,0 +1,2 @@
+После того, как вы взяли в руку синюю щетку, обращается вверх тормашками и вы начинаете увязать в зефире.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.